src/patternfly/components/ExpandableSection/expandable-section.scss

```diff
@@ -13,6 +13,24 @@
   --#{$expandable-section}__toggle-icon--m-expand-top--Rotate: 0;
   --#{$expandable-section}--m-expanded__toggle-icon--Rotate: 90deg;
   --#{$expandable-section}--m-expanded__toggle-icon--m-expand-top--Rotate: -90deg;
+  --#{$expandable-section}__content--TransitionDuration--expand--slide: 0s;
```

Comment:
I'm failing to see how this is needed. I tried adjusting it and removing it and didn't see any difference.

Reply:
It's  part of the default values to keep the slide from happening in reduced motion. By default, `--#{$expandable-section}__content--TransitionDuration--expand--slide` and `--#{$expandable-section}__content--TransitionDuration--collapse--slide` are "0s" (the expand and collapse "slide", aka `translate`, duration), and `--#{$expandable-section}__content--TranslateY` is `0` (the "slide" amount).

For reduced motion "no-preference," those are changed to the motion amounts.